guides/common/modules/con_template-writing-reference.adoc

```diff
@@ -25,7 +25,7 @@ Report templates::
 For more information, see xref:Using_Report_Templates_to_Monitor_Hosts_{context}[].
 
 Templates for partition tables::
-For more information, see {ProvisioningDocURL}creating-partition-tables_provisioning[Creating partition tables] in _{ProvisioningDocTitle}_.
+For more information, see {ProvisioningDocURL}creating-partition-tables-by-using-web-ui[] in _{ProvisioningDocTitle}_.
```

Comment:
```suggestion
For more information, see {ProvisioningDocURL}creating-partition-tables-by-using-web-ui[Creating partition tables by using {ProjectWebUI}] in _{ProvisioningDocTitle}_.
```

IMO every link that uses DocURL needs a title. At least that's how we do it so far.